docs/getting-started/install-configure.md

```diff
@@ -5,44 +5,27 @@ weight: 2
 indent: true
 ---
 
-# Choosing Hosted or Self-Hosted Crossplane
+# Choosing Your Crossplane Distribution
 
 Users looking to use Crossplane for the first time have two options available to
-them today. The first way is to use a hosted Crossplane service like [Upbound
-Cloud][Upbound Cloud]. Alternatively, users looking for some more
-flexibility can install Crossplane into their own Kubernetes cluster.
+them today. The first way option to use the upstream distribution of Crossplane 
+which is maintained and released by the community. 
 
-Crossplane will be installed using the regularly published Helm chart. The Helm
-chart contains all the custom resources and controllers needed to deploy and
-configure Crossplane.
+Your second option is to use a vendor supported Crossplane distribution. These 
```

Comment:
```suggestion
The second option is to use a vendor supported Crossplane distribution. These 
```

Indifferent to "Your" of "The" but just want to be consistent between this sentence and the previous one.